tests/ActivityTest.php

```diff
@@ -43,26 +43,17 @@ public function testInstantiation() {
     }
 
     public function testFromJSONInvalidNull() {
-        $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_NONE
-        );
+        $this->setExpectedException('TinCan\JSONParseErrorException');
```

Comment:
I definitely have a soft spot for something like this:

```php
try {
    throwAnExceptionSomehow();
} catch(YourCustomException $e) {
     $this->assertEquals(1, $e->getCode());
     $this->assertEquals('Message', $e->getMessage());
     return;
}

$this->fail('We should have caught a custom exception and exited this test.');
```

Even though it's far more verbose people can actually just grab the try/catch block and use it themselves.